tests/Froxlor/ValidateTest.php

```diff
@@ -86,12 +86,29 @@ public function testValidateIpCidr()
 		$this->assertEquals("12.34.56.78/24", $result);
 	}
 
+    public function testValidateIpv6Disallowed()
+    {
+        $this->expectException("Exception");
```

Comment:
Please expect a certain Message, only expecting `Exception` is very vague

Reply:
So .. unlike here?! https://github.com/Froxlor/Froxlor/blob/0afbe3d13b67d439177ae23a4f60028b91cc7687/tests/Froxlor/ValidateTest.php#L72

Reply:
Hm must be a reason why I did not specify the exceptionmessage (like in any other test...) cant remember. Let's leave it like that :)